components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -339,10 +340,10 @@ impl ReadStats {
             .entry(region_id)
             .or_insert_with(|| RegionInfo::new(sample_num));
         region_info.update_peer(peer);
-        region_info.add_query_num(kind, query_num);
         if is_read_query(kind) {
             region_info.add_key_ranges(key_ranges);
         }
+        region_info.add_query_num(kind, query_num);
```

Comment:
Why reorder it?

Reply:
to ensure qps is correct when add key range